### ISSUE TITLE
[Roadmap] 1.3.1 Patch release

### ISSUE THREAD

**Issue by hcho3** [dmlc/xgboost#6518]
We are looking to release a patch release 1.3.1, primarily to incorporate pull request #6517. This patch is important to ensure that models from versions earlier than 1.0.0 can be read in the latest XGBoost.

I'd like to know if there are other pull request that should go into 1.3.1.

This issue will remain open until December 21, 2020.

@dmlc/xgboost-committer

Pull requests
- [x] Enable loading model from <1.0.0 trained with objective='binary:logitraw' (#6517)
- [x] Fix handling of print period in EvaluationMonitor (#6499)
- [x] Move metric configuration into booster (#6504)
- [x] Fix `save_best` early stopping option (#6523)
- [x] Remove `cupy.array_equal`, since it's not compatible with cuPy 7.8 (#6528)

**Comment by trivialfis** [comment]
I will extract part of https://github.com/dmlc/xgboost/pull/6506 into 1.3 to fix https://github.com/dmlc/xgboost/issues/6514 .

**Comment by trivialfis** [comment]
@hcho3  I have PRs for all patches needed for back porting.  Hopefully the holiday goes smoothly.  Please let me know if there's anything I can help with the patch release.

**Comment by trivialfis** [comment]
@hcho3 Do you need help on updating the rel page?

**Comment by hcho3** [comment]
@trivialfis I've uploaded the package to PyPI now. If you'd like, why don't you go ahead and put up a new Release entry?

**Comment by trivialfis** [comment]
Let me do that tonight.

**Comment by trivialfis** [comment]
> If you'd like, why don't you go ahead and put up a new Release entry?

Done.  @hcho3 Thanks for steering the patch release.
